x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -566,7 +566,8 @@ export class RulesClient {
       throw Boom.badRequest(`Error creating rule: could not create API key - ${error.message}`);
     }
 
-    await this.validateActions(ruleType, data.actions);
+    const usesGlobalFreqParams = this.usesValidGlobalFreqParams(data);
+    await this.validateActions(ruleType, data.actions, usesGlobalFreqParams);
```

Comment:
Nit: I'd call `usesValidGlobalFreqParams` inside the `validateActions` method. `usesValidGlobalFreqParams` and its result seems to not be used anywhere else.